pkg/tsdb/cloudwatch/request_parser.go

```diff
@@ -257,9 +301,9 @@ func getRetainedPeriods(timeSince time.Duration) []int {
 	}
 }
 
-func parseDimensions(model *simplejson.Json) (map[string][]string, error) {
+func parseDimensions(model QueryJson) (map[string][]string, error) {
```

Comment:
@iwysiu re: https://github.com/grafana/grafana/pull/51062#discussion_r916952900 we don't need to test for float on `Limit` since we now have a type (`LogQueryJson`) that defines the data.